pkg/render/logstorage.go

```diff
@@ -717,9 +713,12 @@ func (es *elasticsearchComponent) nodeSets() []esv1.NodeSet {
 // NodeSet
 func (es *elasticsearchComponent) nodeSetTemplate(pvcTemplate corev1.PersistentVolumeClaim) esv1.NodeSet {
 	config := map[string]interface{}{
-		"node.master":                 "true",
-		"node.data":                   "true",
-		"node.ingest":                 "true",
+		"node.roles": []string{
+			"data",
+			"ingest",
+			"master",
+			"remote_cluster_client",
```

Comment:
Should we remove this role?

Reply:
We might need these role - data, ingest, master same as the previous version.
Reg:remote_cluster_client :  if we are setting node.roles, then [remote_cluster_client](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#node-roles) role must be explicitly included to enable cross-cluster search. Otherwise, this could result in an access denied issue when loading the Stack Monitoring page in Kibana.

Reply:
:/ so be it...